packages/jellyfish-api-core/__tests__/category/wallet.test.ts

```diff
@@ -632,3 +632,185 @@ describe('masternode', () => {
     })
   })
 })
+
+describe('sendMany', () => {
+  // NOTE(sp): defid side(c++) does not have much tests for sendmany RPC atm.
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  // util functions NOTE(sp) : may be move to a common util file if possible ??
+  /**
+   * Returns matching utxos for given transaction id and address.
+   *
+   * @param {string} txId transaction id
+   * @param {string} address address
+   * @return {Promise<UTXO[]>} matching UTXO[]
+   */
+  const getMatchingUTXO = async (txId: string, address: string): Promise<UTXO[]> => {
+    const options: ListUnspentOptions = {
+      addresses: [address]
+    }
+    const matchingUTXOs: UTXO[] = []
+
+    const listUnspent = async (): Promise<void> => {
+      const utxos: UTXO[] = await client.wallet.listUnspent(1, 9999999, options)
+      utxos.forEach(utxo => {
+        expect(utxo.address).toBe(address)
+        if (utxo.txid === txId) {
+          matchingUTXOs.push(utxo)
+        }
+      })
+    }
+    await listUnspent()
+    return matchingUTXOs
+  }
+
+  //= ==============================================================================================
+  it('should getBalance >= 100', async () => {
+    return await waitForExpect(async () => {
+      const balance: BigNumber = await client.wallet.getBalance()
+      expect(balance.isGreaterThan(new BigNumber('100'))).toBe(true)
+    })
+  })
```

Comment:
this is not a part of test
suggestion here is to park this assertion on `beforeAll()`

Reply:
yes. wil do that.

Reply:
@surangap container has much more testing tools available now. You can take a look at those implementations and use them. If you need to add more please create PR for them.

https://github.com/DeFiCh/jellyfish/blob/63fd7700ef9959cc50bb2dadfc3cff202771aaab/packages/testcontainers/src/chains/reg_test_container/masternode.ts#L105-L114